docs/apis/streaming/connectors/kinesis.md

```diff
@@ -299,3 +299,41 @@ Otherwise, the returned stream name is used.
 Other optional configuration keys can be found in `KinesisConfigConstants`.
 		
 		
+### Using non-AWS Kinesis Endpoints
+
+It is sometimes desirable to have Flink operate as a consumer or producer against a non-AWS Kinesis endpoint such as kinesalite; this is especially useful when performing functional testing of a Flink application. The AWS endpoint that would normally be inferred by the AWS region set in the Flink configuration must be overriden via a configuration property.
+
+To override the AWS endpoint, set the `CONFIG_AWS_ENDPOINT` property in the Flink configuration, in addition to the `CONFIG_AWS_REGION` required by Flink. Although the region is required, it will not be used to determine the AWS endpoint URL.
```

Comment:
Previous content in the documentation always have the `KinesisConfigConstants.*` prefix when mentioning config keys. Let's keep consistency with that.